### PR TITLE
INFO-289 [DOCS] New README and Quick-start Guide

### PR DESCRIPTION
**Pull request by TheRealEdDawson**
Info 289/new readme and quickstart guide

### CODE REVIEW

File: LearnosityDemo/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:15545",
```
Comment (rhiannon-eldridge-lrn) [inline LearnosityDemo/Properties/launchSettings.json:6]: I wonder if this means that on windows the port used is different... we should probably test this or change the port to 5000 to keep it consistent with non-windows platforms.

File: docs/quickstart/assessment/standalone-assessment.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
+# Copyright (c) 2021 Learnosity, MIT License
```
Comment (rhiannon-eldridge-lrn) [inline docs/quickstart/assessment/standalone-assessment.py:1]: I assume this file was added accidentally?

File: LearnosityDemo/Program.cs [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace LearnosityDemo
+{
+    public class Program
```
Comment (rhiannon-eldridge-lrn) [inline LearnosityDemo/Program.cs:12]: I suggest adding `static` to the class definition. ie. `public static class Program`

File: LearnosityDemo/Pages/ItemsAPIDemo.cshtml.cs [hunk 4]
```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using LearnositySDK.Request;
+using LearnositySDK.Utils;
+// static LearnositySDK.Credentials;
```
Comment (rhiannon-eldridge-lrn) [inline LearnosityDemo/Pages/ItemsAPIDemo.cshtml.cs:9]: this would be good to fix

File: LearnosityDemo/Pages/Index.cshtml.cs [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    public class IndexModel : PageModel
+    {
+        private readonly ILogger<IndexModel> _logger;
+
+        public IndexModel(ILogger<IndexModel> logger)
+        {
+            _logger = logger;
+        }
+
+        public void OnGet()
```
Comment (rhiannon-eldridge-lrn) [inline LearnosityDemo/Pages/Index.cshtml.cs:20]: try removing this `OnGet` function and see if anything breaks - if it doesn't do anything then we should remove it.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# Contributing
```
Comment (rhiannon-eldridge-lrn) [inline CONTRIBUTING.md:1]: This has references to python and such in it, would be good to update to reference .NET

Could also be good to add a github PR template like we have in some of our other SDKs.

**Review by rhiannon-eldridge-lrn — CHANGES_REQUESTED** [review]
Hey @TheRealEdDawson please let me know if you have any questions about this feedback.

File: docs/quickstart/assessment/standalone-assessment.py (hunk 2 above)
Comment (TheRealEdDawson) [reply docs/quickstart/assessment/standalone-assessment.py:1]: Yes. Sorry I have this approach of putting placeholder things in here, to remind me to put something in that does what it did. I'll remove. :-)

Comment (TheRealEdDawson) [reply docs/quickstart/assessment/standalone-assessment.py]: Removed here: https://github.com/Learnosity/learnosity-sdk-asp.net/commit/0c5500ec37c959b273105a3f13853be0231cca1b

File: LearnosityDemo/Program.cs (hunk 3 above)
Comment (TheRealEdDawson) [reply LearnosityDemo/Program.cs]: Fixed here: https://github.com/Learnosity/learnosity-sdk-asp.net/commit/2ad77f005b6fad299f2bc28e117ce41ebb451b80

File: LearnosityDemo/Pages/Index.cshtml.cs (hunk 5 above)
Comment (TheRealEdDawson) [reply LearnosityDemo/Pages/Index.cshtml.cs]: Runs fine without, so I deleted it. https://github.com/Learnosity/learnosity-sdk-asp.net/commit/4ee05704adf4d448d380f2038232e064e80762d3

File: LearnosityDemo/Pages/ItemsAPIDemo.cshtml.cs (hunk 4 above)
Comment (TheRealEdDawson) [reply LearnosityDemo/Pages/ItemsAPIDemo.cshtml.cs]: Fixed: https://github.com/Learnosity/learnosity-sdk-asp.net/commit/b81074bd4af12796fe00b45f67c0700d1f686d5d

File: LearnosityDemo/Properties/launchSettings.json (hunk 1 above)
Comment (TheRealEdDawson) [reply LearnosityDemo/Properties/launchSettings.json]: Confirmed, Windows uses that port number to launch "IIS Express" on Windows. Tested, and updated to used port 5000 for consistency. https://github.com/Learnosity/learnosity-sdk-asp.net/commit/57c5fffa8793930658fe718805af14626b030b92

**Review by rhiannon-eldridge-lrn — APPROVED** [review]
LGTM!

**Pull request merged**